buildSrc/src/main/groovy/io.micronaut.build.internal.convention-base.gradle

```diff
@@ -14,7 +14,7 @@ repositories {
     mavenCentral()
 }
 
-configurations.all {
+configurations.configureEach {
```

Comment:
This is actually not required. Configurations cannot be configured lazily. Actually using `configureEach` on the configuration container can lead to hard to debug errors, so I would simply revert these changes.

Reply:
Don't always trust your IDE I guess ;)